images/manageiq-base/container-assets/clean_dnf_rpm

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Not for this PR, but I wonder if we should put this script in one of our RPMs and run it on the appliance to reduce image size there too

Reply:
Oh that's an interesting idea - agreed that we can do that in a follow up